app/src/main/java/com/example/playlistmaker2/TrackAdapter.kt

```diff
@@ -1,20 +1,12 @@
 package com.example.playlistmaker2
 
-import android.content.Context
-import android.util.TypedValue
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
-import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
-import com.bumptech.glide.Glide
-import com.bumptech.glide.load.resource.bitmap.RoundedCorners
-import java.text.SimpleDateFormat
-import java.util.Locale
-
 class TrackAdapter(
-    private var trackList: MutableList<Track>) : RecyclerView.Adapter<TrackAdapter.TrackViewHolder>() {
+    private var trackList: ArrayList<Track>) : RecyclerView.Adapter<TrackViewHolder>() {
```

Comment:
```suggestion
    private val trackList: ArrayList<Track>) : RecyclerView.Adapter<TrackViewHolder>() {
```
Здесь тоже мьютабельный список и от мьютабельной ссылки мало смысла